package.json

```diff
@@ -67,7 +67,7 @@
     "eslint-plugin-react": "~7.0.0",
     "eslint-plugin-react-native": "^2.2.1",
     "eslint-plugin-spellcheck": "~0.0.6",
-    "flow-bin": "~0.40.0",
+    "flow-bin": "^0.40.0",
```

Comment:
I think we have to have a strict '0.40' so the `~` was intentional. That is what RN is on.
Did you need the newer ones?